fixtures/clay-paky/show-batten-100.json

```diff
@@ -0,0 +1,756 @@
+    ],
+    "productPage": [
+      "https://www.claypaky.it/en/products/showbatten100"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=2n3CmtSITH0"
+    ]
+  },
+  "physical": {
+    "dimensions": [1000, 100, 186],
+    "weight": 10,
+    "power": 170,
+    "DMXconnector": "5-pin",
+    "bulb": {
+      "type": "10x 15W RGBW LED (Osram Ostar SMT LE RTDUW S2W)",
+      "colorTemperature": 6000,
+      "lumens": 6800
+    },
+    "lens": {
+      "degreesMinMax": [10, 43]
+    },
+    "focus": {
+      "type": "Head",
+      "tiltMax": 240
+    }
+  },
+  "matrix": {
+    "pixelCount": [10, 1, 1]
+  },
+  "availableChannels": {
+    "Red": {
+      "fineChannelAliases": ["Red fine"],
+      "highlightValue": 65535,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "fineChannelAliases": ["Green fine"],
+      "highlightValue": 65535,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "fineChannelAliases": ["Blue fine"],
+      "highlightValue": 65535,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "fineChannelAliases": ["White fine"],
+      "highlightValue": 65535,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "CTO": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 8],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [9, 255],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "8000K",
+          "colorTemperatureEnd": "2500K"
+        }
+      ]
+    },
+    "Color Macro": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 9],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [10, 10],
+          "type": "ColorPreset",
+          "colors": ["#e4000a"],
+          "comment": "Red"
+        },
+        {
+          "dmxRange": [11, 11],
+          "type": "ColorPreset",
+          "colors": ["#00e400"],
+          "comment": "Green"
+        },
+        {
+          "dmxRange": [12, 12],
+          "type": "ColorPreset",
+          "colors": ["#0000e4"],
+          "comment": "Blue"
+        },
+        {
+          "dmxRange": [13, 13],
+          "type": "ColorPreset",
+          "colors": ["#00e4e4"],
+          "comment": "Cyan"
+        },
+        {
+          "dmxRange": [14, 14],
+          "type": "ColorPreset",
+          "colors": ["#e8d100"],
+          "comment": "Yellow"
+        },
+        {
+          "dmxRange": [15, 15],
+          "type": "ColorPreset",
+          "colors": ["#e403e4"],
+          "comment": "Magenta"
+        },
+        {
+          "dmxRange": [16, 16],
+          "type": "ColorPreset",
+          "colors": ["#7fe4e4"],
+          "comment": "White 7000K"
```

Comment:
You can use the `colorTemperature` property here, same with following capabilities.